Source/Core/Core/HW/EXI/BBA/BuiltIn.cpp

```diff
@@ -0,0 +1,774 @@
+// Copyright 2008 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
```

Comment:
Generally when a new file is created, the year should be the current year (so 2022), even if it's been based off of a different file with an earlier date.